frontend/src/lib/components/PropertyFilters/PropertyFilters.tsx

```diff
@@ -38,6 +38,7 @@ interface PropertyFiltersProps {
     propertyAllowList?: { [key in TaxonomicFilterGroupType]?: string[] }
     allowRelativeDateOptions?: boolean
     disabledReason?: string
+    exactMatchFeatureFlagCohortOperators?: boolean
```

Comment:
this feels like _such_ kludge to prop drill this value but it was the easiest way to get this fix out and specify it explicitly for the feature flag release conditions.  It's pretty annoying that we can't do advanced cohort matching for feature flags alone.  Open to ideas on a better way to do this.